### DISCUSSION TITLE
Weird Html chunks when app reloads

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#52049] [Help]
### Summary

Sometimes this error happens especially when reload page
![err](https://github.com/vercel/next.js/assets/125838640/28bbb41b-901f-427c-87a6-3e84b3f910df)

### Additional information

```js
I deployed this app on digital ocean.
```

**Comment by wwwdepot** [comment]
Update to latest version?

**Reply by ghost** [reply]
i think it the latest version next js.additionally i am using API route functionality which is beta
